packages/salesforcedx-vscode-core/src/orgBrowser/metadataCmp.ts

```diff
@@ -227,13 +209,8 @@ export class ComponentUtils {
     );
     return componentList;
   }
-
-  /**
-   * Builds a list of all fields of the standard or custom object from existing json file at the componentsPath
-   * @param componentsPath
-   * @returns a list of all fields of the standard or custom object
-   */
-  public fetchExistingCustomObjectsFields( sObject: string, componentsPath: string) {
+// TODO: fetchExistingCustomObjectsFields requires only one parameter componentsPath and corresponding tests have to be modified
```

Comment:
Is there still something left to do here?

Reply:
Yes, fetchExistingCustomObjectsFields() requires only one parameter i.e. componentsPath. folderName has to be removed. Corresponding changes has to be made in the test file.

Reply:
My vote the would be to remove the `folderName` parameter, remove the TODO, and update the tests in this PR.